library/ui-styles/src/main/res/values/styles_location.xml

```diff
@@ -5,17 +5,23 @@
         <item name="android:foreground">?selectableItemBackground</item>
         <item name="android:background">@android:color/transparent</item>
         <item name="android:textSize">12sp</item>
-        <item name="android:padding">0dp</item>
         <item name="android:gravity">center</item>
+        <item name="android:padding">0dp</item>
+        <item name="android:minWidth">0dp</item>
```

Comment:
Is this necessary while we already set width of the view?

Reply:
Yes this is the tricky part. We want the buttons to respect certain insets on the sides. If we don't have the `minWidth` property, it does not take the insets into account...